### ISSUE TITLE
rtables::as_html duplicates the first row cells when split_cols_by_multivar is used with more than one variable.

### ISSUE THREAD

**Issue by kpagacz** [Roche/rtables#208]
rtables::as_html duplicates the first row cells when split_cols_by_multivar is used with more than one variable.

```r
library(rtables)
lyt <- basic_table() %>%
  split_cols_by("am") %>%
  split_cols_by_multivar(vars = c("disp", "hp"))
result <- build_table(lyt = lyt, df = mtcars)

print(result)
print(rtables::as_html(result))
```

The outputs:
![image](https://user-images.githubusercontent.com/24480246/123417953-bdcd9e00-d5b8-11eb-88e8-060b9ab5fc0f.png)

Notice that the cells in the first row of `result` span two cells of other columns. Meanwhile, rtables::as_html returns an HTML table, in which the first row spans double the number of other rows. In essence, I would expect a result to be this:
```html
...
  <tr class="">
    <th class="" style="white-space:pre;"></th>
    <th class=" text-center" colspan="2">1</th>
    <th class=" text-center" colspan="2">0</th>
  </tr>
...
```

**Comment by kpagacz** [comment]
@waddella @gmbecker

**Comment by gmbecker** [comment]
This was fixed in the above referenced PR and can be closed, correct? @waddella @anajens

**Comment by waddella** [comment]
Yes, this has been fixed with commit 9f116d189ca777a0e65f1bc8a5079f639e1e8279
